segmenters/nlp/SlidingWindowSegmenter/README.md

```diff
@@ -1,3 +1,62 @@
 # SlidingWindowSegmenter
 
-split the text on the doc-level into overlapping substrings on the chunk-level. 
\ No newline at end of file
+SlidingWindowSegmenter is one segmenter used in nlp domain. It splits the text on the doc-level into overlapping substrings on the chunk-level. 
```

Comment:
Maybe also "the NLP" here to keep it consistent with the other PRs?

Reply:
👍  Thanks!